pipeline/filters/log-metrics.md

```diff
@@ -0,0 +1,147 @@
+---
+description: Generate metrics from logs
+---
+
+# Log Metrics
+
+The _Log Metrics Filter_ plugin allows you to generate log-derived metrics. It currently supports modes to count records, sum up field values over a record stream or provide a gauge for field values. You can also match or exclude specific records based on regular expression patterns for values or nested values. This filter plugin does not actually act as a record filter and does not change or drop records.
```

Comment:
> This filter plugin does not actually act as a record filter and does not change or drop records.

What does it do then? Pass on the logs as-is and then create new records in the metrics pipeline? Please make this more clear.